tests/gordo_components/serializer/test_serializer_into_definition.py

```diff
@@ -126,46 +126,47 @@ def test_pipeline_into_definition(self):
         expected_definition = (
             """
             sklearn.pipeline.Pipeline:
+              memory:
                 steps:
                     - sklearn.decomposition.pca.PCA:
-                        n_components: 2
                         copy: true
-                        whiten: false
-                        svd_solver: auto
-                        tol: 0.0
                         iterated_power: auto
+                        n_components: 2
                         random_state:
+                        svd_solver: auto
+                        tol: 0.0
+                        whiten: false
                     - sklearn.pipeline.FeatureUnion:
+                        n_jobs:
                         transformer_list:
                         - sklearn.decomposition.pca.PCA:
-                            n_components: 3
                             copy: true
-                            whiten: false
-                            svd_solver: auto
-                            tol: 0.0
                             iterated_power: auto
+                            n_components: 3
                             random_state:
+                            svd_solver: auto
+                            tol: 0.0
+                            whiten: false
                         - sklearn.pipeline.Pipeline:
+                            memory:
                             steps:
                             - sklearn.preprocessing.data.MinMaxScaler:
-                                feature_range:
-                                - 0
-                                - 1
                                 copy: true
+                                feature_range:
+                                  - 0
+                                  - 1
                             - sklearn.decomposition.truncated_svd.TruncatedSVD:
-                                n_components: 2
                                 algorithm: randomized
+                                n_components: 2
                                 n_iter: 5
                                 random_state:
                                 tol: 0.0
-                            memory:
                             verbose: false
-                        n_jobs:
                         transformer_weights:
                         verbose: false
                     - gordo_components.model.models.KerasAutoEncoder:
+                        build_fn:
```

Comment:
It is a bit odd that we pass this one around, but we "explicitly" ignore it in our constructor in `KerasAutoEncoder`. I see how it comes there, the call to `params = super().get_params(**params)` in `get_params` provides it. Should we remove it before we return in `get_params`, or just ignore this whole "problem"?

Reply:
I have no problem removing it, certainly something which is used "internally", but the only potential problem is if some gamy data scientist starts playing around with parameters within their notebook and are passing functions directly. I think @Natalie-Caruana was doing this at one point; passing the builder function directly to the model

Reply:
But we already dont support that really, since the constructor of `KerasAutoEncoder` completly ignores any `build_fn` parameter

Reply:
ah, so right, I was thinking about how you can pass `kind` a function. I'll update to remove the `build_fn` :+1: